specs/triggers/triggerbinding.spec

```diff
@@ -0,0 +1,70 @@
+# Verify triggerbindings spec
+
+Pre condition:
+  * Validate Operator should be installed
+
+## Verify CEL marshaljson function Test: PIPELINES-06-TC01
```

Comment:
The test cases with ID `PIPELINES-06` series already present, use `PIPELINES-10-TC01` and add `PIPELINES-10` in the first line of the file

Reply:
in that case shall I change it 09 ?? instead of 10